devdocs/agents.md

```diff
@@ -0,0 +1,29 @@
+# AGENTS.md
+
+## Setup and build
+
+- Clone and build: `git clone https://github.com/nushell/nushell && cd nushell && cargo build`
+- Run from source: `cargo run` or `use toolkit.nu; toolkit run`
+- Use `toolkit.nu` for all dev tasks: `use toolkit.nu`
```

Comment:
This expects that the agent is running nushell. I don't think we can expect that to always be the case. Perhaps we should try something like:
```
nu -c "use toolkit.nu; toolkit check pr"
```

Though that would expect that nu is in the path.

Reply:
Feel free to push any changes to the PR. I was just trying to get something documented as a starting place.